olive/extra_dependencies.json

```diff
@@ -31,5 +31,22 @@
     ],
     "torch-tensorrt": [
         "torch-tensorrt"
+    ],
```

Comment:
nit: Will this too detailed? how about lora and lora-ort? Also please added it in lora example's readme.